packages/grafana-toolkit/docker/grafana-plugin-ci-alpine/scripts/deploy.sh

```diff
@@ -18,16 +18,16 @@ apk add --no-cache curl npm yarn build-base openssh git-lfs perl-utils coreutils
 # apk add --no-cache xvfb glib nss nspr gdk-pixbuf "gtk+3.0" pango atk cairo dbus-libs libxcomposite libxrender libxi libxtst libxrandr libxscrnsaver alsa-lib at-spi2-atk at-spi2-core cups-libs gcompat libc6-compat
 
 # Install Go
-filename="go1.17.8.linux-amd64.tar.gz"
-get_file "https://dl.google.com/go/$filename" "/tmp/$filename" "980e65a863377e69fd9b67df9d8395fd8e93858e7a24c9f55803421e453f4f99"
+filename="go1.18.linux-amd64.tar.gz"
```

Comment:
i think these two need updates as well since they are used by the drone extension:
* `packages/grafana-toolkit/docker/grafana-plugin-ci/scripts/deploy.sh`
* `packages/grafana-toolkit/docker/grafana-plugin-ci-e2e/scripts/deploy.sh`

Reply:
I don't think it's necessary since those images don't do the build, but I can update those as well.

Reply:
ah ok, i was trying to follow this, and i assumed [this step](https://github.com/grafana/grafana-drone-extension/blob/2b2716752987b92b9a8fc827a380c6082f8bc8e0/pluginjson/pipeline-ci.go#L73-L84) used [grafana-plugin-ci](https://github.com/grafana/grafana-drone-extension/blob/main/pluginjson/pipeline-ci.go#L15). do you know where `grafana-plugin-ci-alpine` gets included? i'm not seeing it in search results for that repo

Reply:
i see what's happening now. it looks like it's just a separate tag unlike the e2e image